string name
string[] nodes
bool isActive
bool isRunning
NodeLaunchParam[] launchparams
